src/nvim/window.c

```diff
@@ -3296,8 +3296,14 @@ void tabpage_move(int nr)
   tabpage_T *tp;
   tabpage_T *tp_dst;
 
-  if (first_tabpage->tp_next == NULL)
+  // cannot happen (to silence static analysis)
+  if (curtab == NULL) {
```

Comment:
If it can't happen, `assert(curtab);` is understood to mean that. Then the comment isn't needed, either.